index_countdown.html

```diff
@@ -66,7 +66,7 @@
 
 		<!-- ## SLIDE TEXT ## -->
 		<div id="qcSlideText">
-			<h1 id="slidecaption">Write/Speak/Code 2017 is here</h1>
+			<h1 id="slidecaption">Write/Speak/Code 2017 conference is here</h1>
 			<p id="slidedesc">Join us for 4 intense days of workshops, seminars, and talks. Let's build the blueprint that will take your career to the next level!</p>
```

Comment:
And add a `<br>` or something so that each of these sentences is on a separate line